export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+  filename = osp.join(osp.dirname(osp.abspath(__file__)), 'params.yaml')
+  with open(filename, 'r') as f:
+    params = yaml.safe_load(f)
+  # TODO(samarth): access params stuff by attributes
+  export_dir = osp.expanduser(osp.join(args.scene_path, params['common']['GLTF_SCENE_DIR_NAME']))
+  os.makedirs(export_dir, exist_ok=True)
+  include_actors = args.actors.split(',') if args.actors else None
+  
+  filename = osp.join(export_dir, 'actors_information.json')
+  if osp.isfile(filename):
+    with open(filename, 'r') as f:
+      body_properties = json.load(f)
+  else:
+    body_properties = {}
+  
+  # Iterate through the actors
+  for actor in unreal.GameplayStatics.get_all_actors_of_class(world, unreal.Actor):
+    if isinstance(actor, unreal.CameraActor) or isinstance(actor, unreal.SceneCapture2D) or \
+      isinstance(actor, unreal.PointLight):
+      continue
+    actor_name = actor.get_actor_label()
+    if 'HDRIBackdrop' in actor_name:
+      continue
+    if include_actors and (actor_name not in include_actors):
+      continue
+    
+    this_body_properties = {'geoms': {}, }
+    # check if actor has geometry
+    has_geometry = False
+    i = 0
+    for c in actor.get_components_by_class(unreal.StaticMeshComponent):
+      static_mesh = c.get_editor_property('static_mesh')
+      if static_mesh is None:
+        print(f'actor {actor_name} component {c.get_name()} does not have a static mesh')
+      elif static_mesh.get_name() != 'SM_Dummy':
+        # TODO(samarth) get MeshComponent elements from UE component
+        c_name = c.get_name()
+        if c_name == 'StaticMeshComponent0':
+          c_name = actor_name
+        this_body_properties['geoms'][c_name] = MeshComponent('coacd', i)._asdict()
+        has_geometry = True
+        i += 1
+
+    if has_geometry:
+      export_meshes(actor, export_dir)
+    else:
+      print(f'Actor {actor_name}, does not have geometry')
+      continue
+    
+    if isinstance(actor, unreal.StaticMeshActor):
+      this_body_properties['moving'] = actor.get_editor_property('static_mesh_component').get_editor_property('body_instance').get_editor_property('simulate_physics')
+    else: # TODO(samarth) get the actual flag for the actuated body
+      this_body_properties['moving'] = False
+    this_body_properties['root_component_name'] = actor.get_editor_property('root_component').get_name()
+    body_properties[actor_name] = this_body_properties
+    
+    constraint_dicts = []
+    for component in actor.get_components_by_class(unreal.ActorComponent):
+      if not isinstance(component, unreal.PhysicsConstraintComponent):
+        continue
+      
+      r = component.get_editor_property('relative_rotation')
+      rot = unreal.Rotator(r.roll, r.pitch, r.yaw)
+      
+      parent = str(component.get_editor_property('component_name2').get_editor_property('component_name'))
+      if (parent == 'DefaultSceneRoot') or (parent == 'StaticMeshComponent0'):
+        parent = actor_name
```

Comment:
Is this special case making our lives any easier?

Reply:
Please see this comment: https://github.com/isl-org/spear/pull/254#discussion_r1477598198.